src/ai/susi/server/api/susi/SusiService.java

```diff
@@ -21,30 +21,14 @@
 
 import ai.susi.DAO;
 import ai.susi.json.JsonObjectWithDefault;
-import ai.susi.mind.SusiArgument;
-import ai.susi.mind.SusiCognition;
-import ai.susi.mind.SusiMind;
-import ai.susi.mind.SusiSkill;
-import ai.susi.mind.SusiThought;
-import ai.susi.server.APIException;
-import ai.susi.server.APIHandler;
-import ai.susi.server.AbstractAPIHandler;
-import ai.susi.server.Authorization;
-import ai.susi.server.Query;
-import ai.susi.server.ServiceResponse;
-import ai.susi.server.UserRole;
-
+import ai.susi.mind.*;
+import ai.susi.server.*;
```

Comment:
Using wildcards for imports is a _BAD_ idea.

Reply:
I agree completely. It's also bad that this is a mixed change together with the fix. But lets accept this by now to move on. Please @yashLadha do not use wild cards in the future.